root/etc/cont-init.d/12-samples

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/with-contenv bash
+
+# remove old samples
+find /config/nginx/ -name "*.conf.sample" -type f -delete
+
+# copy new samples
+cp \
```

Comment:
Can replace everything with
`cp -fR /defaults/nginx /config/nginx`

Reply:
That would overwrite users `dhparams.pem` which is handled here:
https://github.com/linuxserver/docker-baseimage-alpine-nginx/pull/84/files#diff-f874e5986500c1c68f4a87ce8acfff8a61e109d0ecee3197e593a9a34459360dR38-R43

We don't want to replace that if a user puts their own file. We only want `*.conf.sample` files. If there's a clever way to do that recursively I'm happy to make this more compact.